external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/JsonUtils.scala

```diff
@@ -95,4 +95,20 @@ private object JsonUtils {
     }
     Serialization.write(result)
   }
+
+  /**
+   * Write per-topic partition lag as json string
+   */
+  def partitionLags(latestOffsets: Map[TopicPartition, Long],
+                    processedOffsets: Map[TopicPartition, Long]): String = {
+    val result = new HashMap[String, HashMap[Int, Long]]()
```

Comment:
nit: `HashMap.empty[String, HashMap[Int, Long]]`

Reply:
Had followed the style in other parts of the class. Addressed and refactored the other places as well.